run_task.sh

```
@@ -0,0 +1,10 @@
+
+strace_file=`date -u +"%Y-%m-%dT%H:%M:%SZ"`
+#strace -f -o logs/strace_${strace_file}.txt -s 9999
+~/gemini-cli/bundle/gemini.js --output-format json \
+			      --approval-mode yolo \
+			      --model gemini-2.5-flash \
+			      --checkpointing \
+			      --include-directories ~/nix \
+			      --debug \
+			      $@ 2>&1 | tee logs/stdout_${strace_file}.txt 
```

Shellcheck (0.11.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)